pkg/controller/download_request_controller.go

```diff
@@ -178,20 +180,22 @@ func (c *downloadRequestController) processNextWorkItem() bool {
 // processDownloadRequest is the default per-item sync handler. It generates a pre-signed URL for
 // a new DownloadRequest or deletes the DownloadRequest if it has expired.
 func (c *downloadRequestController) processDownloadRequest(key string) error {
-	glog.V(4).Infof("processDownloadRequest for key %q", key)
+	logContext := c.logger.WithField("key", key)
+
+	logContext.Info("Running processDownloadRequest")
 	ns, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
-		glog.V(4).Infof("error splitting key %q: %v", key, err)
+		logContext.WithField("error", err).Info("Error splitting key")
```

Comment:
return errors.Wrap(err, "cannot split key"), assuming that cache.SplitMetaNamespaceKey doesn't return a good error. If it does, just return err, no need to log as well.